main.css

```diff
@@ -0,0 +1,107 @@
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  height: 92px;
```

Comment:
- [ ] Great job using flexbox. However, by giving a width it's affecting the responsiveness of the website by creating unnecessary space when the width is more than 375px I suggest removing the fixed width or giving max-width.

### Your Impleme
![image](https://user-images.githubusercontent.com/59416136/158661926-6eb830d6-bcd2-4e8d-b78d-4e0a73efcd2a.png)
ntation